lib/new_relic/agent/instrumentation/active_record_helper.rb

```diff
@@ -208,14 +215,10 @@ module InstanceIdentification
           PRODUCT_SYMBOLS = {
             "mysql" => :mysql,
             "mysql2" => :mysql,
-            "mysql2_makara" => :mysql,
```

Comment:
Same deal here, the `bare_adapter_name` chokepoint method will be used instead of having every suffix based name defined.